deploy/chart/templates/0000_50_olm_08-catalog-operator.deployment.yaml

```diff
@@ -81,12 +81,12 @@ spec:
             httpGet:
               path: /healthz
               port: {{ .Values.catalog.service.internalPort }}
-              scheme: {{ if and .Values.catalog.tlsKeyPath .Values.catalog.tlsCertPath }}HTTPS{{ else }}HTTP{{end}}
+              scheme: {{ if or .Values.olm.tlsSecret .Values.catalog.clientCASecret }}HTTPS{{ else }}HTTP{{end}}
```

Comment:
shouldn't this be `.Values.catalog.tlsSecret`?

Reply:
Yes - sorry working late at night.

Reply:
Addressed in the latest version of the pr